telegraf.conf.tpl

```diff
@@ -250,3 +277,26 @@
 # [[inputs.netstat]]
 #   # no configuration
 
+# Read metrics from Kafka topic(s)
+{% if INPUT_KAFKA_ENABLED == "true" %}
+[[inputs.kafka_consumer]]
+  ## topic(s) to consume
+  topics = ["telegraf"]
+  ## an array of Zookeeper connection strings
+  zookeeper_peers = ["zookeeper:2181"]
```

Comment:
not sure how it is used, but clearly we need a INPUT_KAFKA_ZOOKEEPER_PEER env var